rocky/reports/templates/partials/report_types_selection.html

```diff
@@ -36,42 +36,43 @@ <h2>
                 {% include "summary/ooi_selection.html" %}
 
             {% endif %}
-        {% endif %}
-        <form novalidate
-              method="post"
-              action="{{ previous }}"
-              class="inline layout-wide">
-            {% csrf_token %}
-            {% include "forms/report_form_fields.html" %}
+            <form novalidate
```

Comment:
Is this novalidate needed?

Reply:
I mean also Manon styling for showing form errors

Reply:
If validation can already be done by the browser / before submitting the form in my opinion that it is better user experience then having it done after the form is submitted. If there are styling issues with the validation errors we should fix those syling issues instead of not doing the validation.